service_thing_aggregate/src/main/scala/controllers/ObjectAggregationController.scala

```diff
@@ -0,0 +1,40 @@
+package controllers
+
+import com.google.inject.Inject
+import models.MusitResults._
+import play.api.Logger
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.libs.json.Json
+import play.api.mvc._
+import services.{ ObjectAggregationService, StorageNodeService }
+
+import scala.concurrent.Future
+
+class ObjectAggregationController @Inject() (
+    service: ObjectAggregationService,
+    storageNodeService: StorageNodeService
+) extends Controller {
+
+  def getObjectsIfNodeExists(nodeId: Long) = Action.async { request =>
```

Comment:
Somewhat redundantly named, or? Wouldn't get `getObjects` suffice?
If the node with nodeId doesn't exists it directly implies that there are noe objects :-)